src/js/all/stories/CustomThemed/ExtendTheme.tsx

```diff
@@ -13,7 +13,7 @@ const custom: ThemeType = {
   },
   anchor: {
     extend: (props) => css`
-      color: ${props.href ? 'green' : 'red'};
+      color: ${props['href'] ? 'green' : 'red'};
```

Comment:
Assuming this was the linter. Did we have a rule change on us?

Reply:
This didn't come from upgrading any lint dependencies. This change came from upgrading typescript.

Reply:
After upgrading I got the error `TS2339: Property 'href' does not exist on type 'never'.`